portfolio/src/main/webapp/script.js

```diff
@@ -53,3 +53,27 @@ function getHelloNzingaUsingAsyncAwait() {
       });
     });
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Animal');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Brazil!', 10],
+          ['Mexico!', 5],
+          ['Alaska!', 15]
+        ]);
+
+  const options = {
+    'title': 'Destinations',
+    'width':500,
+    'height':400
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma): Include a trailing comma whenever there is a line break between the final property and the closing brace.

Feel free to address this in the other PR instead that depends on these changes!